controllers/API/announcement.js

```diff
@@ -51,7 +51,7 @@ router.get("/API/announcement", function (req, res) {
 	Announcement
```

Comment:
This should actually change to:
```
const count = req.query.count; // Possibly parse this to an integer first. Unsure; see comment below.
delete req.query.count;

Announcement
		.find(req.query)
 		.sort({ $natural: -1 })
	        .limit(count || 200)
```

This is a mistake I introduced in my previous PR and it breaks the count functionality on all GET requests. It will now return 0 entries when you pass a count parameter.

Reply:
Alright am gonna change it to that.